### PR TITLE
Move all png optimizations in a loop, so everything is in parallel there.

### PR DESCRIPTION
**Pull request by stefanbeller**

### CODE REVIEW

**Comment by stefanbeller** [comment]
do not pul this yet, I need to correct it first :P

File: image_optim.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 #TODO: also use pngout ?
 
 echo "To kill the script, run"
-echo "while true ;do ; killall optipng ; killall advpng ; killall pngcrush ; done"
+echo "while true ; do killall optipng ; killall advpng ; killall pngcrush ; done"
```
Comment (matthiaskrgr) [inline image_optim.sh:5]: Hmm, was there something not working?
I think the previous version always worked for me.

Comment (stefanbeller) [inline image_optim.sh:5]: The inital version did not work for me, kept saying wrong syntax, and I got it working with removing that semicolon.

File: image_optim.sh [hunk 2]
```diff
@@ -59,74 +59,49 @@ jpeg_remove_comment_and_exiv()
 	timestart
 	git ls-files ./ | grep -e "\.jpg$" -e "\.jpeg" | xargs -P ${cpucores} -n 1 jpegoptim --strip-all >> /tmp/image_optim_jpeg.log
 	timeend
+	wait
```
Comment (matthiaskrgr) [inline image_optim.sh:62]: I don't understand why we need the "wait" here.

**Pull request merged**